src/v/kafka/protocol/schemata/generator.py

```diff
@@ -1126,32 +1126,28 @@ class response;
 {% macro conditional_tag_encode(tdef, vec) %}
 {%- if tdef.is_array %}
 {%- if tdef.nullable() %}
-{%- call tag_version_guard(tdef) %}
 if ({{ tdef.name }}) {
```

Comment:
can you make commit message more descriptive ?

Reply:
Done